recipes-bsp/imx-seco/imx-seco_3.7.1.bb

```diff
@@ -23,4 +23,5 @@ do_deploy () {
     install -m 0644 ${S}/firmware/seco/${SECO_FIRMWARE_NAME} ${DEPLOYDIR}
 }
 
-COMPATIBLE_MACHINE = "(mx8qm|mx8qxp|mx8qxpc0|mx8phantomdxl|mx8dxl)"
```

Comment:
This is bad for tracking of what is compatible, please keep the old way.

Reply:
ok, that's fine
note that `imx-seco-libs_git` is using already the simplified rule

Reply:
back to list rule ..I'll add also the rest of machines

Reply:
Good; please replace the simplified rule, so we keep it consistent.